Package.swift

```diff
@@ -1,9 +1,15 @@
 import PackageDescription
 
 let package = Package(
-  name: "SnowMan",
-  dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
-  ]
+  	name: "SnowMan",
+  	targets: [
+  		Target(name: "Dictionary"),
+  		Target(name: "SnowManLogic", dependencies: ["Dictionary"]),
```

Comment:
try and find out what the error is telling you, and update your package based on that.